cli/pazuzu/actions/config.go

```diff
@@ -0,0 +1,61 @@
+package actions
+
+import (
+	"errors"
+	"fmt"
+	"github.com/urfave/cli"
+	"github.com/zalando-incubator/pazuzu"
+	"github.com/zalando-incubator/pazuzu/config"
+	"os"
+	"text/tabwriter"
+)
+
+func ConfigShow(c *cli.Context) error {
+	if c.NArg() != 0 {
+		return errors.New("Wrong number of arguments")
```

Comment:
I think in separate one, since it's not related to the original PR purpose. What do you think?